source/extensions/common/wasm/wavm/wavm.cc

```diff
@@ -215,15 +215,15 @@ Wavm::~Wavm() {
 }
 
 std::unique_ptr<WasmVm> Wavm::clone() {
-  auto wavm = new Wavm();
+  auto wavm = std::make_unique<Wavm>();
   wavm->compartment_ = WAVM::Runtime::cloneCompartment(compartment_);
   wavm->memory_ = WAVM::Runtime::remapToClonedCompartment(memory_, wavm->compartment_);
   wavm->context_ = WAVM::Runtime::createContext(wavm->compartment_);
   wavm->envoyModuleInstance_ =
       WAVM::Runtime::remapToClonedCompartment(envoyModuleInstance_, wavm->compartment_);
   wavm->moduleInstance_ =
       WAVM::Runtime::remapToClonedCompartment(moduleInstance_, wavm->compartment_);
-  return std::unique_ptr<WasmVm>(wavm);
+  return wavm;
```

Comment:
I'll leave this one to @jplevyak, since he wrote this code, and this change is more about the style than anything else.

Reply:
get it
CC @PiotrSikora

Reply:
Can we just do std::make_unique and then return the result?

Reply:
Ah, didn't see the whole change.  LGTM